src/js/components/Menu.js

```diff
@@ -149,6 +160,13 @@ var MenuDrop = React.createClass({
     if (this.props.small) {
       classes.push(CLASS_ROOT + "__drop--small");
     }
+    if (this.props.className) {
+      var classNames = this.props.className.split(" ");
+      classNames = classNames.map(function(className) {
+        return className + "__drop";
```

Comment:
Hmm, I was debating about this too but I'm not sure what's a good solution for this. It felt weird to only append `__drop` to one of the classes instead. I thought it would work better to append `__drop` to all of the classes that are passed to the Menu component since these are basically a set of new classes anyway.